src/node_buffer.cc

```diff
@@ -310,6 +310,70 @@ void Base64Slice(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+// bytesCopied = buffer.mask(mask, target, targetOffset, sourceStart, sourceEnd)
+void Mask(const FunctionCallbackInfo<Value> &args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  uint32_t mask32 = args[0]->Uint32Value();
+  char* mask = (char*)&mask32;
```

Comment:
Yes, a pointer to char may alias a pointer to another type (but not the other way around.)